preprocessy/scaling/_main.py

```diff
@@ -38,6 +38,9 @@ def __init__(
         self.__validate_input()
         self.final_df = None
 
+    def __repr__(self):
+        return f"Scaler(df=None,type=None,columns=None,is_combined=False,critical_value={0})"
```

Comment:
Replace defaults for all params with `self.param_name` except for the dataframe param.